src/components/Navbar.tsx

```diff
@@ -8,17 +17,16 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink className={getLinkActive} to="/">
             Home
-          </a>
+          </NavLink>
 
-          <a
-            aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+          <NavLink
+            className={getLinkActive}
+            to={{ pathname: 'people', search: searchParams.toString() }}
```

Comment:
The `to` prop in `NavLink` is using an object with `pathname` and `search`. While this is valid, ensure that this format is necessary for your routing setup. If not, consider using a simple string path if it suffices.